src/components/InPageNavigation/InPageNavigation.stories.tsx

```diff
@@ -0,0 +1,64 @@
+import React from 'react'
+import { InPageNavigation } from './InPageNavigation'
+import { CONTENT } from './content'
+import { HeadingLevel } from '../../types/headingLevel'
+
+export default {
+  title: 'Components/In-Page Navigation',
+  component: InPageNavigation,
+  argTypes: {
+    headingLevel: {
+      control: 'select',
+      options: ['h1', 'h2', 'h3', 'h4', 'h5', 'h6'],
+    },
+    rootMargin: {
+      control: 'text',
+    },
+    scrollOffset: {
+      control: 'text',
+    },
+    threshold: {
+      control: { type: 'range', min: 0, max: 1, step: 0.01 },
+    },
+    title: {
+      control: 'text',
+    },
+  },
+  args: {
+    headingLevel: 'h4',
+    rootMargin: '0px 0px 0px 0px',
+    scrollOffset: '0rem',
```

Comment:
**question(scrollOffset)**: USWDS defines this as 

> Number of pixels scroll should offset from the top of the parent element.

Should we expect anything other than a flat number of pixels then?

```suggestion
    scrollOffset: '0',
```

Reply:
Either way, `0px`, `0rem`, etc. are best written as `0` for clarity, so I recommend making this change even if nothing else is updated to expect numeric values

Reply:
I personally would prefer the flexibility in spacing units rather than strictly to px, which this approach still supports. I did change the story to use just "0", though.

Should we write up a DR on implementation details like this and the offset decision above? My approach is generally to start with the original implementation and then deviate only if improves eng experience or accessibility and is still compatible with the original implementation.